ChangeLog.d/cipher-delayed-output.txt

```diff
@@ -0,0 +1,6 @@
+API changes
+   * For multi-part AEAD operations with the Cipher module, calling
```

Comment:
Minor: we sometimes write module names in all caps (mostly when they're normally all-caps things, like AES and CTR_DRBG) and sometimes write them in lowercase, but we don't use a leading cap, so
```suggestion
   * For multi-part AEAD operations with the cipher module, calling
```